### DISCUSSION TITLE
Images don't seem to stitch, despite pretty clear matches

### DISCUSSION THREAD

**Discussion by redfast00** [OpenStitching/stitching#207] [Q&A]
I'm trying to stitch together a long schematic that I scanned page by page. I've managed to narrow it down to two pages that don't seem to want to stitch together, despite looking like they should (look at the text at the bottom of the page). I've tried the four different detectors, but this does not seem to work. Am I missing some parameters that are especially helpful for black/white scanners?

```
docker container run --rm -v .:/data openstitching/stitch
```

![page-0](https://github.com/OpenStitching/stitching/assets/10746993/7a0377e6-936c-4b18-a940-dfe78cdae6ab)
![page-1](https://github.com/OpenStitching/stitching/assets/10746993/c2b6cc44-c67e-432b-9909-fb90c258ef02)

**Comment by lukasalexanderweber** [comment]
Yes, you definitely need the `--affine` parameter
